Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -268,17 +268,16 @@ public static function initializeSystemLogger(Bootstrap $bootstrap)
      * Initialize the exception storage
      *
      * @param Bootstrap $bootstrap
+     * @param array $settings The Neos.Flow settings
      * @return ThrowableStorageInterface
      * @throws FlowException
      * @throws \Neos\Flow\Configuration\Exception\InvalidConfigurationTypeException
      */
-    protected static function initializeExceptionStorage(Bootstrap $bootstrap): ThrowableStorageInterface
+    protected static function initializeExceptionStorage(Bootstrap $bootstrap, array $settings): ThrowableStorageInterface
     {
-        $configurationManager = $bootstrap->getEarlyInstance(ConfigurationManager::class);
-        $settings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow');
-
         $storageClassName = $settings['log']['throwables']['storageClass'] ?? FileStorage::class;
         $storageOptions = $settings['log']['throwables']['optionsByImplementation'][$storageClassName] ?? [];
+        $renderRequest = $settings['log']['throwables']['renderRequestInformation'] ?? true;
```

Comment:
If this way, lets name the variable "renderRequestInformation" to honour the full Settings key